python/tank/authentication/interactive_authentication.py

```diff
@@ -36,9 +36,17 @@
 import sys
 import os
 
+try:
+    from .ui.qt_abstraction import QtCore, QtGui, QtNetwork, QtWebKit
+except Exception:
+    QtCore = None
```

Comment:
please add some comments to explain when this happens? would it make more sense if `.ui.qt_abstraction` returned None values in the case of no import? I *think* that's how it's working in `sgtk.platform`.

Reply:
This was something done following a discussion with JF: I moved some imports which were within a method/function up to the top of the file, try/expect and all.

JF tells me that the try/except are no longer required. Should the import from qt_abstraction fail, the offending module(s) will be None.

I will remove the try/except and add a comment to that effect.